src/easypost.js

```diff
@@ -126,6 +126,17 @@ export default class API {
     this.use(RESOURCES);
   }
 
+  static copyApi(api, options = {}) {
```

Comment:
What is this? If we absolutely need it then we need to document it and potentially name it something more concrete.